src/utils/FireboltSdkManager.js

```diff
@@ -0,0 +1,292 @@
+    const [majorA, minorA, patchA] = parseVersion(versionA);
+    const [majorB, minorB, patchB] = parseVersion(versionB);
+
+    if (majorA !== majorB) {
+      return majorA < majorB;
+    }
+    if (minorA !== minorB) {
+      return minorA < minorB;
+    }
+    return patchA < patchB;
+  }
+}
+
+// ----------------------------------------
+// 2. SDK JSON Loader
+// ----------------------------------------
+
+/**
+ * Dynamically loads SDK JSON definitions from OpenRPC files.
+ */
+class SdkJsonLoader {
+  /**
+   * Initializes the loader with context for each SDK type.
+   * @param {Object} sdkContexts - Contexts for dynamically loading OpenRPC JSON files.
+   */
+  constructor(sdkContexts) {
+    this.sdkContexts = sdkContexts;
+  }
+
+  /**
+   * Loads the JSON definitions for SDKs.
+   * @param {Object} sdkPaths - A map of SDK paths.
+   * @returns {Object} A map of loaded JSON definitions.
+   */
+  loadJson(sdkPaths) {
+    const sdkJson = {};
+    for (const [sdkType, sdkPath] of Object.entries(sdkPaths)) {
+      try {
+        const sdkModule = this.sdkContexts[sdkType](`./${sdkPath.split('/').pop()}`);
+        sdkJson[sdkType] = sdkModule;
+      } catch (error) {
+        console.warn(`Failed to import module for SDK type '${sdkType}': ${error.message}`);
+      }
+    }
+    return sdkJson;
+  }
+}
+
+// ----------------------------------------
+// 3. SDK Module Mapper
+// ----------------------------------------
+
+/**
+ * Maps SDK module names to their corresponding JSON definitions.
+ */
+class SdkModuleMapper {
+  /**
+   * Maps modules to SDK types.
+   * @param {Object} sdkJson - A map of loaded JSON definitions.
+   * @param {Object} sdkImports - The imported SDK modules.
+   * @returns {Object} A map of SDK modules categorized by type.
+   */
+  static mapModules(sdkJson, sdkImports) {
+    const sdkModules = {};
+    const sdkModulesMap = {};
+
+    // Set of PascalCase module names
+    const pascalCaseModules = new Set(['SecondScreen', 'SecureStorage', 'ClosedCaptions', 'AcknowledgeChallenge', 'DeveloperTools', 'LifecycleManagement', 'PinChallenge', 'UserGrants', 'VoiceGuidance']);
+
+    for (const [sdkType, json] of Object.entries(sdkJson)) {
+      // Check if 'x-module-descriptions' exists and has keys, otherwise use extractUniqueModules
+      const moduleNames = json.info && json.info['x-module-descriptions'] && Object.keys(json.info['x-module-descriptions']).length > 0 ? Object.keys(json.info['x-module-descriptions']) : extractUniqueModules(json.methods);
+
+      sdkModules[sdkType] = moduleNames;
+      sdkModulesMap[sdkType] = moduleNames.reduce((acc, module) => {
+        // Find a matching PascalCase module
+        const matchingPascalModule = Array.from(pascalCaseModules).find((pascalModule) => pascalModule.toLowerCase() === module.toLowerCase());
+
+        // Use the PascalCase module if there's a match; otherwise, use the original module
+        const moduleKey = matchingPascalModule || module;
```

Comment:
Why do we need to do this special handling for pascal modules?

Reply:
Older versions of the sdk/openrpc json do not use the same PascalCasing as the imports. The issue with this is when I try to dynamically import them. 

For example:
`import { Secondscreen } from '@firebolt-js/manage-sdk`
is an incorrect import that gives issue.

PascalCasing of specific modules is required so the import is correct:
`import { SecondScreen } from '@firebolt-js/manage-sdk`